libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -545,6 +545,36 @@ bool AP_AHRS_NavEKF::use_compass(void)
     return AP_AHRS_DCM::use_compass();
 }
 
+// return the quaternion defining the rotation from NED to XYZ (body) axes
+bool AP_AHRS_NavEKF::get_quaternion(Quaternion &quat) const
+{
+    switch (active_EKF_type()) {
+    case EKFType::NONE:
+        return false;
+#if HAL_NAVEKF2_AVAILABLE
+    case EKFType::TWO:
+        EKF2.getQuaternion(-1, quat);
+        return _ekf2_started;
```

Comment:
Is this the correct check?  No filter status bit we could look at?

Reply:
I think it's correct or at least consistent with what AP_AHRS_NavEKF::get_secondary_attitude() does which is the equivalent for euler angle attitude.